src/System.Security.Cryptography.Csp/tests/RSACryptoServiceProviderBackCompat.cs

```diff
@@ -116,6 +122,63 @@ public static void SignHash_PublicKey()
             }
         }
 
+        [Fact]
+        public static void OldVerifyHash()
+        {
+            byte[] dataHash, signature;
+
+            using (HashAlgorithm hash = SHA256.Create())
+            {
+                dataHash = hash.ComputeHash(TestData.HelloBytes);
+            }
+
+            // Use the new RSA SignHash method to generate the signature
+            using (RSA rsa = RSAFactory.Create())
```

Comment:
Nothing else in this file uses RSAFactory.Create, go ahead and just use `new RSACryptoServiceProvider` here.